client/src/containers/AssetTitle/view.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Row from '@components/Row';
 
-const AssetTitle = ({ title }) => {
+const AssetTitle = ({ title, editable, claimId, name }) => {
   return (
     <Row>
-      <h3>{title}</h3>
+      <h3>
+        {title}
+        {editable && (<span> (<Link to={`/edit/${claimId}/${name}`}>edit</Link>)</span>)}
```

Comment:
better to have this out of the h3 ("edit" isn't part of the title for SEO / accessibility purposes, etc.)